src/roswire/ros2/ros2.py

```diff
@@ -5,8 +5,8 @@
 
 import attr
 
-from .launch import ROS2LaunchManager
-from .node_manager import ROS2NodeManager
+from .manager import ROS2NodeManager
```

Comment:
`.manager` should remain as `.node_manager` to avoid ambiguity with other managed resources (e.g., services)